force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -252,15 +277,21 @@ public with sharing class FlowControlLoopsCollections {
      * @return The sum of positive integers from the list.
      */
     public static Integer sumPositiveIntegers() {
-        List<Integer> numbers = new List<Integer>{-1, 2, -3, 4, -5, 6}; //DO NOT CHANGE
-
-        // Loop through the list of integers
-            // if the number is negative skip this iteration
-        
-
-        return null; // Replace null with the variable you used to store the result
+        List<Integer> numbers = new List<Integer>{-1, 2, -3, 4, -5, 6};
+        List<Integer> finalsumList = new List<Integer>(); 
+        Integer totalSum = 0; 
+        for(Integer num : numbers){
+            if ( num > 0){
+                finalsumList.add(num);
```

Comment:
Rather than creating a second list to hold the positive numbers, you can do the addition to totalSum inside this if statement